### PR TITLE
check recurring logic cli keys

### PR DESCRIPTION
**Pull request by pondrejk**
Adding a simple extension to an existing test to cover for 2129432

### CODE REVIEW

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py -k test_positive_run_reccuring_job

File: tests/foreman/cli/test_remoteexecution.py [hunk 1]
```diff
@@ -650,6 +654,15 @@ def test_positive_run_reccuring_job(self, rex_contenthost):
         rec_logic = RecurringLogic.info({'id': result['recurring-logic-id']})
         assert rec_logic['state'] == 'finished'
         assert rec_logic['iteration'] == '2'
+        # 2129432
+        rec_logic_keys = rec_logic.keys()
+        assert 'action' in rec_logic_keys
+        assert 'last-occurence' in rec_logic_keys
```
Comment (ColeHiggins2) [inline tests/foreman/cli/test_remoteexecution.py:660]: Looks like prt job is failing due to spelling. should be 'last-occurrence'

File: tests/foreman/cli/test_remoteexecution.py [hunk 2]
```diff
@@ -650,6 +654,15 @@ def test_positive_run_reccuring_job(self, rex_contenthost):
...
+        assert 'next-occurence' in rec_logic_keys
```
Comment (ColeHiggins2) [inline tests/foreman/cli/test_remoteexecution.py:661]: same here

**Review by ColeHiggins2 — CHANGES_REQUESTED** [review]

File: tests/foreman/cli/test_remoteexecution.py (hunk 1 above)
Comment (pondrejk) [reply tests/foreman/cli/test_remoteexecution.py:660]: facepalm

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py -k test_positive_run_reccuring_job

**Review by shweta83 — APPROVED** [review]
Ack

**Review by shubhamsg199 — APPROVED** [review]

**Review by ColeHiggins2 — APPROVED** [review]

**Review by rplevka — APPROVED** [review]

**Pull request merged**